python/get_my_watchlist_feed.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+
+"""
+    get_my_watchlist.py
```

Comment:
File name differs here..

Reply:
Fixed in https://github.com/srish/MediaWiki-Action-API-Code-Samples/pull/42/commits/6d0834180dc5c959cb016f363a1c0eb046357ba5